layouts/_default/baseof.html

```diff
@@ -124,6 +124,13 @@
 
         <!-- set the color theme -->
         <script src="/js/themes_init.min.js"></script>
+
+        <!-- noindex meta tag -->
+        {{ if .Site.Data.args.preliminary }}
+        <meta name="robots" content="noindex">
+        {{ else if .Site.Data.args.archive }}
+        <meta name="robots" content="noindex">
+        {{ end }}
```

Comment:
This is the same content, you don't need to include it twice.  Use `{{ if or }}`